### ISSUE TITLE
config.name with a colon breaks on iOS

### ISSUE THREAD

**Issue by rdurfee** [renpy/renpy#3197]
If the game's config.name contains a colon character (":"), the game will not start on iOS. The Xcode project builds fine, but when run it produces an error:

`
ImportError: "No module named site"
`

**Comment by renpytom** [comment]
Can you post any more of the error? I don't see an obvious way it could be related, but iOS is weird. You can probably get around this by setting build.directory_name to something different, without the :.

**Comment by rdurfee** [comment]
The game starts, showing the launch screen, then it quits. It's not actually an in-game error, but an output message, displayed in the All or Package Output window.

Setting the directory_name like this has no effect.
```
define config.name = _("Test: The Sequel")
define build.directory_name = "TestTheSequel"
```

**Comment by rdurfee** [comment]
I don't know if it's related, but when it creates the xcodeproject file, it names it with a / character instead of a colon....

`Test/ The Sequel.xcodeproj`

**Comment by renpytom** [comment]
Fixed by renpy/renpy-build@df89ba03abea4466c9e286c5df6e2db870301224 .
